kubemarine/procedures/check_paas.py

```diff
@@ -239,8 +239,7 @@ def check_packages_versions(cluster: KubernetesCluster, tc: TestCase, hosts_to_p
             if version.startswith(package_name):
                 good_results.append(version)
             else:
-                bad_results.append(version)
-
+                bad_results.append(f"{package} (Actual version: {version}, Expected package version: {package_name})")
```

Comment:
I got `Expected package version: containerd=1.6.` that looks strange (note trailing dot at the end). Expected version is `package`, not `package_name`.